content/docs/guides/neon-authorize-troubleshooting.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Neon Authorize Troubleshooting
+subtitle: Common issues and solutions when using Neon Authorize
+enableTableOfContents: true
+---
+
+This page covers common errors you might encounter when implementing Row-level Security policies with Neon Authorize and your authentication provider.
+
+---
+
+## Password authentication issues
```

Comment:
I'd much rather have the error message as the "indexable" thing here. I think it makes a lot more sense for people to just have a list of "possible errors" and then immediately be able to read information about that error.

This is how the original google doc that I wrote is structured.

Reply:
got it - fixed